pages/integrations/amazon_eventbridge.md

```diff
@@ -449,6 +449,8 @@ AWS EventBridge has strict limits on the size of the payload as documented in [A
   }
 }
 ```
+>📘
```

Comment:
It's best to include a title for notes and tips like this. Maybe:

```suggestion

>📘 Check job result
```

^ Also add a blank line to separate the element from the code block.